microsoft-edge/web-platform/sidebar.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Develop for the sidebar
+description: Learn to develop experiences for the sidebar in Microsoft Edge, by either installing a Progressive Web App, or using the extension sidebar API.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 07/24/2023
+---
+# Develop for the sidebar
+
+The sidebar in Microsoft Edge is a persistent pane located on the side of the browser, which coexists with the primary content of the browser. The sidebar allows users to easily access popular websites and utilities alongside their browser tabs. The content in the sidebar augments the user's primary task by enabling side-by-side browsing and minimizing the need to switch contexts between browser tabs. With the sidebar in Microsoft Edge, users can access the productivity tools they need while staying in their workflow.
+
+![The sidebar in Microsoft Edge, showing useful tools, alongside the main tab](./sidebar-images/sidebar.png)
+
+As a developer, you can leverage the sidebar for your own experiences in two ways:
+
+* By adapting your existing Progressive Web App (PWA) to run in the sidebar.
+* By making use of the Sidebar API in your Microsoft Edge extension.
+
+
+<!-- ====================================================================== -->
+## Choose the right approach
+
+The two different ways in which you can leverage the sidebar are designed to meet different developer needs. The following table summarizes the key differences between the two approaches:
+
+|Approach | Sidebar app | Sidebar extension |
+|---|---|---|
+| Definition | A Progressive Web App (PWA) that runs in the sidebar. | A Microsoft Edge extension that displays content in the sidebar. |
+| Distribution | Like a website, the app is hosted on the app's web server. | The extension is distributed through Microsoft Edge Add-ons. |
+| Capabilities | The app can make use of all web APIs and features, as well as PWA capabilities such as offline support, push, or file system access. | The extension can make use of powerful extension APIs, such as reading or modifying the current page. |
+| Context | The app runs in its own web context, separate from the main tab. | The extension can access the current page and modify it via its content script. |
```

Comment:
```suggestion
| Context | The app runs in its own web context, separate from the main tab. | The extension can access the current webpage and modify the webpage via the extension's content script. |
```
* Prefer direct wording instead of "it" and "its" indirection pointers.
* Tech review "its": Is it the extension's content script, or the webpage's content script?